actions/class.TestContent.php

```diff
@@ -120,33 +120,4 @@ public function download() {
         
         tao_helpers_Http::returnFile($path);
     }
-    
-    /**
```

Comment:
Should this method really be deleted?

Reply:
according to this spike https://oat-sa.atlassian.net/browse/TAO-6784